src/main/java/no/nav/pto/veilarbportefolje/controller/EnhetController.java

```diff
@@ -96,4 +88,26 @@ public EnhetTiltak hentTiltak(@PathVariable("enhet") String enhet) {
 
         return tiltakService.hentEnhettiltak(EnhetId.of(enhet));
     }
+
+    @GetMapping("/{enhet}/foedeland")
+    public String hentFoedeland(
+            @PathVariable("enhet")
+            String enhet) throws JsonProcessingException {
+        ValideringsRegler.sjekkEnhet(enhet);
+        authService.tilgangTilEnhet(enhet);
+
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Is this necessary? 
Wont spring boot fix this if you return the value directly?